### PR TITLE
LRCI-1398 Append include globs to appropriate modified modules and ch…

### PR DESCRIPTION
**Pull request by brittneyq**
…eck for modules in paths

https://issues.liferay.com/browse/LRCI-1398

### CODE REVIEW

File: modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/test/clazz/group/ModulesJUnitBatchTestClassGroup.java [hunk 1]
```diff
@@ -129,11 +131,10 @@ protected List<JobProperty> getRelevantIncludesJobProperties() {
 			return super.getRelevantIncludesJobProperties();
 		}
 
-		List<File> modifiedFilesList =
-			portalGitWorkingDirectory.getModifiedFilesList();
+		Set<File> modifiedModuleDirsList = new HashSet<>();
```
Comment (pyoo47) [inline modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/test/clazz/group/ModulesJUnitBatchTestClassGroup.java:134]: @brittneyq modifiedModuleDirsList is a Set, not a List.

Comment (brittneyq) [reply modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/test/clazz/group/ModulesJUnitBatchTestClassGroup.java:134]: oops sorry I'll make that change

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
ci:test:relevant

**Comment by pyoo47** [comment]
ci:forward

**Pull request closed without merging**